content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+# Sourcegraph 4.0
+
+> NOTE: This post is very WIP. It's a rough draft of what individual teams intend to ship.
+
+- [From code search toward a code intelligence platform](/blog/code-search-to-code-intelligence) - the longer-term vision/story
+- [Secure, easy, and scalable Sourcegraph Cloud managed instances for the enterprise](/blog/enterprise-cloud) - the Cloud announcement (including SOC2 Type 2, etc.) that coincides with 4.0
+
+## TL;DR
+
+`TODO(sqs): The section headers with emojis are what I think we should use, but the wording of the bullet points within are WIP`
+
+##### 🌅 Code intelligence: more useful, more times daily for every dev
+
+- The fastest and most scalable code search (search perf, page load perf, scalability: # repos, big monorepos, # code hosts, etc.)
+- TODO how to introduce the simple UI?
+- Smarter code search
+- High-level aggregations of search results
+- Precise code navigation for 9 languages (adding Ruby and Rust) with auto-indexing
+
+##### 🏗️ High-leverage ways to improve your entire codebase
+
+- Run batch changes server side
+- Something about the new packaging with batch changes and code insights
+
+##### ☁️ Enterprise Cloud dedicated(/managed?) instances
+
+- [Secure, easy, and scalable Sourcegraph Cloud dedicated(/managed?) instances for the enterprise](/blog/enterprise-cloud)
+
+##### 💖 For admins
```

Comment:
@iskyOS @lcolston08 - are we going to mention multi-version upgrades in the release post?

Reply:
Same. It should be listed as a key feature under the Love for admins theme. So this would mean higher up in our features under that theme.